### DISCUSSION TITLE
[Bug] Exported notes don't contain Zotero Tags in YAML header

### DISCUSSION THREAD

**Discussion by cinemakinoeye** [windingwind/zotero-better-notes#855] [Note Templates]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest Zotero and the latest plugin?

- [X] I have confirmed I'm using the latest Zotero and the latest plugin

### Environment

OS: MacOS: 14.2.1  
Zotero Version: 6.0.30
Plugin Version: 1.0.4

### Describe the bug

When using the ExportMDFileHeaderV2 template, the Tags are missing on export.

### Debug Output

When using the ExportMDFileHeaderV2 template, the Tags are missing on export, e.g. 
```
---
tags: []
parent: 'Foucault (notes)'
collections:
    - 'Media Ecology'
version: 16556
libraryID: 1
itemKey: LI7E2GEK
---
```
Even though the parent item for the note has Tags associated with it. I was expecting to see Tags in the exported file, like 
```
---
tags:
    - critcalTheory
    - Foucault
    - mediaTheory
    - power
parent: 'Foucault (notes)'
collections:
    - 'Media Ecology'
version: 16556
libraryID: 1
itemKey: LI7E2GEK
```

### Anything else?

I think there is a simple fix (which I've tried for myself and it works for me). Change the line:

```
header.tags = noteItem.getTags().map((_t) => _t.tag);
```
to 
```
header.tags = noteItem.parentItem.getTags().map((_t) => _t.tag);
```

**Comment by cinemakinoeye** [comment]
One thing I want to add: yes, I acknowledge that Zotero notes can have their tags, so whether you want to export the tags associated with the note or the parent item tags, or both, depends on one's workflow.

**Comment by windingwind** [comment]
https://github.com/windingwind/zotero-better-notes/discussions/694

**Comment by windingwind** [comment]
The tags here are the tags of the note item itself, not it's parent item. You'll need to modify the template.

**Comment by windingwind** [comment]
Oh, I notice you have figured out the solution. I'll convert this thread to a discussion

**Reply by cinemakinoeye** [reply]
I find it odd that the exported notes by default don't inherit the tags from the parent item since it would be awkward to have to add tags to each note, which is why I reported it as a bug rather than an enhancement request. With other integrations, the tags from the parent item comes across when you bring items from Zotero into other note systems.
